src/transaction/log_recovery_redo_parallel.cpp

```diff
@@ -925,9 +884,6 @@ namespace cublog
 	    job->~redo_job_impl ();
```

Comment:
There is no need to explicitly call any of this dtors anymore. They were being called because of the placement ctor call (which, might have allocated extra memory as part of the objects inner workings - not the case in this situation, but added for completitude).

Reply:
done